lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
@ilyavolodin I confirmed the problem for globally installed ESLint.
So.... to fix this route cause, do we needs `=== "node_modules"` check as same as the first way? But it's rejected :confused: